ios/Podfile.lock

```diff
@@ -678,4 +678,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 801f9c43586733b267138576ff038f2dee20112b
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.10.0
```

Comment:
Can you please also update your [CocoaPods installation](https://guides.cocoapods.org/using/getting-started.html) to version 1.10.1.